frontend/src/marketplace/EntryList.tsx

```diff
@@ -32,23 +43,36 @@ export default function EntryList({
       onSelectedChipsChange={onSelectedChipsChange}
       data={data}
       index={index}
-      style={style}
+      style={{ padding: '20px', ...style }}
     />
   )
 
+  const columnWidths = entries.map((entry) => (entry.tags.length === 0 ? 100 : 140))
```

Comment:
Can we pull this outside of Row and into the main component so we can `useMemo` it?

Reply:
I've added it as a useMemo, but it's already outside of `Row`